src/app/view/perspectives/pivot/wrapper/pivot-perspective-wrapper.component.scss

```diff
@@ -1,5 +1,9 @@
 :host {
   display: flex;
   flex-grow: 1;
-  height: 100%;
+  min-height: 100%;
+}
+
+pivot-table {
+  overflow: auto;
```

Comment:
It is not a good idea to defined style selectors based on HTML element names. Especially if it is a name of a component which can be easily changed during some refactoring.

Moreover, in this particular case you can use `overflow-auto` class from Bootstrap.